### PR TITLE
Removes Destructibles

### PR DESCRIPTION
**Pull request by Takahiru**
### Intent of your Pull Request

Refactors some code for adam
destructibles are archaic goodbye destructibles
#### Changelog

:cl:  
tweak: removes destructibles
/:cl:

### CODE REVIEW

**Review by patpol4 — APPROVED** [review]
Maps seem fine, its just line changes, no map edits.

**Comment by JamieD1** [comment]
Looks good, tested? @Takahiru

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
See the following:
https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md#no-duplicated-code

Use object oriented methods for this, which is what destructible was doing

File: code/game/objects/structures.dm [hunk 1]
```diff
@@ -27,6 +30,20 @@
...
+			if(islist(debris))
+				for(var/I in debris)
+					for(var/i in 1 to debris[I])
+						new I (get_turf(src))
+		if(break_message)
+			visible_message(break_message)
+		if(break_sound)
+			playsound(src, break_sound, 50, 1)
+	qdel(src)
+	return 1
```
Comment (Amelia0010) [inline code/game/objects/structures.dm]:
```suggestion
	return TRUE
```

File: code/game/objects/structures.dm [hunk 2]
```diff
@@ -27,6 +30,20 @@
...
+	if(!disassembled)
+		if(!(flags_1 & NODECONSTRUCT_1))
+			if(islist(debris))
+				for(var/I in debris)
+					for(var/i in 1 to debris[I])
+						new I (get_turf(src))
+		if(break_message)
+			visible_message(break_message)
+		if(break_sound)
+			playsound(src, break_sound, 50, 1)
```
Comment (Amelia0010) [inline code/game/objects/structures.dm]:
```suggestion
			playsound(src, break_sound, 50, TRUE)
```

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
Why though?

File: code/game/objects/structures.dm (hunk 1 above)
Comment (Takahiru) [reply code/game/objects/structures.dm]: i didnt think of fixing the original badcode

**Review by Amelia0010 — APPROVED** [review]
There's legitimately no reason to do this

**Comment by Amelia0010** [comment]
Code is fine but I don't see the point, if Jamie or Alexkar disagree with this they can just re-open and merge it immediately

**Pull request closed without merging**